boards/lolin_s3_mini_pro.json

```diff
@@ -0,0 +1,47 @@
+{
+    "build": {
+      "arduino": {
+        "ldscript": "esp32s3_out.ld",
+        "memory_type": "qio_qspi"
+      },
+      "core": "esp32",
+      "extra_flags": [
+        "-DBOARD_HAS_PSRAM",
+        "-DARDUINO_LOLIN_S3_MINI",
+        "-DARDUINO_USB_MODE=1"

```

Comment:
The S3 can run in `USB_MODE = 1` **or** `USB_MODE = 0`. Therfore `-D ARDUINO_USB_MODE=1` should be removed and set on demand in the project's `platformio.ini`. 

Since there is no USB-2-Serial Chip, the extra_flag `-D ARDUINO_USB_CDC_ON_BOOT` should be set to `1`. This will "route" all `Serial.print...` to the correct port.